OpenRA.Mods.Common/Effects/LineImpactProjectile.cs

```diff
@@ -0,0 +1,256 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using OpenRA.Effects;
+using OpenRA.GameRules;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Graphics;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Effects
+{
+	public class LineImpactProjectileInfo : IProjectileInfo
```

Comment:
This name doesn't really fit with our other ones (`BulletInfo`, `MissileInfo`, `LaserInfo`).  How about `BeamInfo`?